### ISSUE TITLE
iPad Dialog doesn't handle target=_blank

### ISSUE THREAD

**Issue by TinoDidriksen** [OfficeDev/office-js#220]
In Word 2.16 for iPad (reinstalled today), in a dialog, links and forms with `target="_blank"` do not open externally in Safari - such links are opened in the same view, regardless of JS tricks tried to force a new window.

The add-in sidebar handles `_blank` just fine and launches external Safari for those links, so it is puzzling why dialogs don't.

## Context
We use dialogs to pop up configuration, because we have quite a lot of things to configure which really doesn't fit in the narrow sidebar. Links to settings' explanation and help should open externally, and they do on all other platforms than iOS. This is causing the add-in to fail your review process, but it doesn't seem possible for us to affect how links are opened from a dialog.

## Your Environment
* Platform: iOS
* Host: Word
* Office version number: 2.16
* Operating System: iOS 10.3.3 (older iPad MD511KN/A)

**Comment by lliu113** [comment]
@TinoDidriksen hey Tino, so sorry for the late response. Is this issue still reproing for you? If so, the easiest way for us to test is by using your add-in manifest. Is this something you're able to provide? 

Thanks!

**Comment by TinoDidriksen** [comment]
Appears this has been fixed in the meantime, either in iOS or in Word. A testcase https://retmig.dk/gas/ojs220/ojs220.xml ( https://retmig.dk/gas/ojs220/ ) works as I want on an iOS 13.3 iPad with newest Word app. The `target="_blank"` link from the popup now opens in an external browser, as it should. This did not happen at the time I filed the bug.

On my old iOS 10.3 iPad it exhibits the broken behavior, but that's historic at this point.

Closing as fixed.
